assets/TWO_VIEW_ESTIMATOR.md

```diff
@@ -8,4 +8,73 @@
 - **Two View Estimator**
 - [Multiview Optimizer](assets/MULTIVIEW_OPTIMIZER.md)
 
-## What is a Two-View Estimator?
\ No newline at end of file
+## What is a Two-View Estimator?
+
+
+Two-View Estimator (TVE) takes information about two images and tries to determine their relative pose (how one camera is positioned and oriented with respect to the other one). It also generates correspondences between keypoint in the images.
+
+As seen in the diagram above, TVE sits between [Correspondence Generator](assets/CORRESPONDENCE_GENERATOR.md) and [Multiview Optimizer](assets/MULTIVIEW_OPTIMIZER.md). TVE does three things in GTSfM:  
```

Comment:
minor nitpick: this can be an unordered list, remove "3 things" as well. I think the exact number depends on how you break it down, and is not important.